proposals/proposal-0001.md

```diff
@@ -0,0 +1,223 @@
+```yaml
+authors: [ "John Doe <john@example.com>", "Jane Doe <jane@example.com>" ]
+```
+
+If the email address is included, and
+
+```yaml
+authors: [ "John Doe", "Jane Doe" ]
+```
+
+If the address is not given.
+
+The `created` header records the date that the proposal was published to the
+Helm project on Github. The header should be in [RFC 3339][rfc3339] format, e.g.
+2020-08-13.
+
+Proposals will typically have a `helm-version` header which indicates the
+version of Helm that the feature will be released with. proposals without a
+`helm-version` header indicate interoperability standards that will initially be
+supported through external libraries and tools, and then supplemented by a later
+proposal to add support to the project's standard library or Command Line
+Interface. Informational and process proposals do not need a `helm-version`
+header.
+
+Proposals may have a `requires` header, indicating the proposal numbers that
+this proposal ends on.
+
+Proposals may also have a `superseded-by` header indicating that a proposal has
+been rendered obsolete by a later document; the value is the number of the
+proposal that replaces the current document. The newer proposal must have a
+`replaces` header containing the number of the proposal that it rendered
+obsolete.
+
+## Auxiliary files
+
+proposals may include auxiliary files such as diagrams. Such files must be named
+proposal-XXX-YY.ext, where "XXX" is the proposal number, "YY" is a serial number
+(starting at 01), and "ext" is replaced by the actual file extension (e.g.
+"png").
+
+## Transferring proposal ownership
+
+It occasionally becomes necessary to transfer ownership of proposals to a new
+champion. In general, it is preferable to retain the original author as a
+co-author of the transferred proposal, but that's really up to the original
+author. A good reason to transfer ownership is because the original author no
+longer has the time or interest in updating it or following through with the
+proposal process, or has fallen off the face of the earth (i.e. is unreachable
+or not responding to email). A bad reason to transfer ownership is because the
+author doesn't agree with the direction of the proposal. One aim of the proposal
+process is to try to build consensus around a proposal, but if that's not
+possible, an author can always submit a competing proposal.
+
+If you are interested in assuming ownership of a proposal, you can also do this
+via pull request. Fork the helm/community repository, make your ownership
+modification, and submit a pull request. You should also send a message asking
+to take over, addressed to both the original author and the org maintainers
+(`@helm/org-maintainers`). If the original author doesn't respond to email in a
+timely manner, the org maintainers will make a unilateral decision (it's not
+like these decisions can't be reversed).
+
+[cncf-helm]: mailto:cncf-helm@lists.cncf.io
+[markdown]: https://daringfireball.net/projects/markdown/syntax
```

Comment:
Are we using the original syntax or common mark (basically what GitHub uses)? If it's common mark we might want to link there.